src/ComplexMethod3.java

```diff
@@ -0,0 +1,163 @@
+public class ComplexMethod3 {
+
+    public String iasdntToEnglishValue(int i) {
+        /*dfg dfgdsf gdsfg dsfg sdf gdsgds*/
+        if (i == 1) {
+            return "One";
+        }
+        if (i == 2) {
+            return "Two";
+        }
+        if (i == 3) {
+            return "Three";
+        }
+        if (i == 4) {
+            return "Four";
+        }
+        if (i == 5) {
+            return "Five";
+        }
+        if (i == 6) {
+            return "Six";
+        }
+        if (i > 6) {
+            throw new IllegalArgumentException(String.format("Invalid input i=%d", i));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Invalid input i=%d" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)